tests/drivers/i2c/i2c_target_api/boards/max32680evkit_max32680_m4.overlay

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2024 Analog Devices, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&i2c0 {
+	status = "okay";
+	pinctrl-0 = <&i2c0a_scl_p0_10 &i2c0a_sda_p0_11>;
```

Comment:
Would it make sense to mark the commonly-accessible I2C marked as `"status" = "okay";` in the board dts?

That would allow anyone just having to build with CONFIG_I2C=y to enable the functionality vs having to introduce an additional dts overlay

Reply:
Yes, good catch

Reply:
Should I create another PR for this?

Reply:
> Should I create another PR for this?

I can volunteer to follow up on that

Reply:
Thanks.